lesson01/assignment/fourFunctions.js

```diff
@@ -0,0 +1,127 @@
+// Code that requires the DOM be loaded should not be run until the DOM is loaded
+document.addEventListener("DOMContentLoaded", function(){
+
+  // Find the funcion-1 form, and handle its submit event
+  const exampleForm = document.getElementById("function-1")
+
+  // Add event listner to submit event
+  exampleForm.addEventListener('submit', function(event){
+    
+    // Prevent the form from trying to submit to a server
+    event.preventDefault()
+
+    // An array to be manipulated
+    let data = [120, 270, 35]
+
+    // Get user input and parse for int
+    const inputField = document.querySelector('#function-1 .input')
+    const userInput = inputField.value
+    const userNumber = parseInt(userInput)
+
+    // Perform a mod of each number in data array with user input and return result
+    const result = data.map( x =>  x % userNumber);
+
+    // Set inner html text to manipulated array
+    document.querySelector('#function-1 .output').innerHTML = result.toString();
+
+    // Console log the manipualted array
+    console.log(`Template literals make things so easy! Here is the result of the prior calculation ${result}`);
+
+    // Clear the form field so the user can try again
+    inputField.value = ''
+  })
+})
+
+// Code that requires the DOM be loaded should not be run until the DOM is loaded
+document.addEventListener("DOMContentLoaded", function(){
```

Comment:
You only need one callback function for the `"DOMContentLoaded"` event. And you can put all of your handlers in the one.